app-header-layout/app-header-layout.html

```diff
@@ -55,52 +55,49 @@
     <style>
       :host {
         display: block;
-      }
 
-      :host([has-scrolling-region]) {
+        /**
+         * Force app-header-layout to have its own stacking context so that its parent can
+         * control the stacking of it relative to other elements (e.g. app-drawer-layout).
```

Comment:
it would nice to mention here that this can be done using `isolation: isolate;`, but it's not fully supported across browsers